lambdas/service/Makefile

```diff
@@ -18,6 +18,7 @@ deploy: state wheels
 	python $(azul_home)/scripts/manage_chalice_deployed.py service upload
 
 delete: state
+    python $(azul_home)/scripts/delete_lambda_iam_role.py service
```

Comment:
Makefiles must use a tab character for indenting lines in a recipe. This looks like four spaces. Did you actually test this? My `make` fails if I use spaces instead of tabs.